### ISSUE TITLE
WorkflowState giving the wrong enum value

### ISSUE THREAD

**Issue by kinow** [cylc/cylc-ui#695]
**Describe the bug**

See comment in GScan PR's review for more: https://github.com/cylc/cylc-ui/pull/543#discussion_r660253772

See TBD mark in WorkflowIcon.vue.

![image](https://user-images.githubusercontent.com/304786/123891758-7fb3ee00-d9ad-11eb-9ec3-3c9ea4989cb5.png)

**Release version(s) and/or repository branch(es) affected?**

master after GScan PR is merged.

**Steps to reproduce the bug**

Run a workflow that receives deltas updating its status. The icon displayed is incorrect.

**Expected behavior**

**Screenshots**

**Additional context**

**Pull requests welcome!**
This is an Open Source project - please consider contributing a bug fix
yourself (please read `CONTRIBUTING.md` before starting any work though).

**Comment by kinow** [comment]
Duplicate of #702
